docs/user-guide/preflight-checks.md

```diff
@@ -21,11 +21,14 @@ This is an example of a ROSflight pre-flight checklist.  You will likely need to
 - Wire-wiggle test (wiggle all wires to look for bad connections)
 - Parameter Check (if using a fixed wing, there are about 8 parameter you will need to change from default  )
 
-### After Powering Up Motors (MAKE SURE THE FLIGHT CONTROLLER IS DISARMED!!!)
+### After Powering Up Motors
+
+!!! warning
+    Be sure flight controller is disarmed before powering up motors!!!
 - Arm/Disarm test
 - Propeller spin test (check directions and response to stick inputs)
 - Control surface test (Fixed Wing)
-- Response to offboard controls (Fixed Wing)
+- Response to offboard controls
 
 
 ## ROSplane Checklist
```

Comment:
Should we remove ROSplane from this documentation? It can have its own. (doesn't have to be in this merge request, just making a note so I don't forget)